### ISSUE TITLE
Update Gale  Chinese recipe

### ISSUE THREAD

**Issue by jtrmal** [kaldi-asr/kaldi#1163]
I just noticed the Kaldi recipe for Chinese Gale uses only the Phase2 data, however there is a lot of additional data -- this is a list I compiled from LDC (I might have unintentionally skip something)
```
LDC2013S08 GALE Phase 2 Chinese Broadcast News Speech                     126hrs 
LDC2013S04 GALE Phase 2 Chinese Broadcast Conversation Speech             120hrs
LDC2014S09 GALE Phase 3 Chinese Broadcast Conversation Speech Part 1      126hrs
LDC2015S06 GALE Phase 3 Chinese Broadcast Conversation Speech Part 2      112hrs
LDC2015S13 GALE Phase 3 Chinese Broadcast News Speech                     150hrs
LDC2016S03 GALE Phase 4 Chinese Broadcast Conversation Speech             172hrs
```
i.e. around 800hrs in total. Perhaps the recipe should be updated? 
y.

**Comment by danpovey** [comment]
@naxingyu, do you have any time to work on this?

**Comment by naxingyu** [comment]
Yes. But I don't have the data...

**Comment by naxingyu** [comment]
Just for record... The corresponding transcriptions are
```
LDC2013T20 GALE Phase 2 Chinese Broadcast News Transcripts 
LDC2013T08 GALE Phase 2 Chinese Broadcast Conversation Transcripts 
LDC2014T28 GALE Phase 3 Chinese Broadcast Conversation Transcripts Part 1 
LDC2015T09 GALE Phase 3 Chinese Broadcast Conversation Transcripts Part 2
LDC2015T25 GALE Phase 3 Chinese Broadcast News Transcripts 
LDC2016T12 GALE Phase 4 Chinese Broadcast Conversation Transcripts 
```

**Comment by jtrmal** [comment]
BTW, I'm working on gale mandarin, I'll make PR hopefully next week.
The recipe is more on the messy side :(
y.

**Comment by naxingyu** [comment]
OK. Then I stop on my side.

X.

**Comment by danpovey** [comment]
Xingyu, I think it's likely that what you're working on and the changes
Yenda will make are different enough that you can just go ahead... but up
to you.

**Comment by naxingyu** [comment]
I'm working on using the full set of Gale Mandarin in that recipe, as 
what is intended to be done in this issue. @jtrmal does it overlap with 
(or included in) what you are doing?

X.

**Comment by jtrmal** [comment]
I'll try to publish whatever might be useful today's evening/tmrw mornign.
My changes weren't related to the nnet3 -- mostly to the data directories
building...
I have also noticed, that the original way how the dev set was generated
was incorrect, as it was a sample of 300 utterances from the training data.
The utterances itself weren't used, but the speakers, the audio channel and
so on were used... So I generated dev sets also.

@naxingyu <https://github.com/naxingyu>, sorry to keep you hanging, but I
needed the system for our LORELEI eval quite quickly, so I've figured out
I'l do it  myself.
y.
